lib/slack_api.rb

```diff
@@ -0,0 +1,23 @@
+class SlackApi
+  def self.send_to_slack(message, channel = nil)
+    body = {
+      text: message,
+      username: 'bub'
+    }
+    body[:channel] = channel if channel
+    slack_http_request(body)
+  end
+
+  def self.slack_http_request(body)
+    uri = URI.parse(SLACK_URL)
+    http = Net::HTTP.new(uri.host, uri.port)
+    http.use_ssl = true
+
+    request = Net::HTTP::Post.new(uri.request_uri)
+    request['Content-Type'] = 'application/json'
+    request.body = body.to_json
+
+    http.request(request)
+  end
```

Comment:
Would recommend [making this method private](http://stackoverflow.com/a/4953043/240427).